relay-dynamic-config/src/defaults.rs

```diff
@@ -105,6 +105,13 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
         & RuleCondition::eq("span.description", APP_START_ROOT_SPAN_DESCRIPTIONS);
 
     [
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/usage@none".into(),
+            field: None,
+            condition: Some(duration_condition.clone()), // same as for count_per_op
```

Comment:
@phacops this condition is open for debate: Does it make sense to count a span as "used" (and thereby create an accepted outcome for it) even if it's not used in any current module? Or should we rather use the condition from the `exclusive_time` metric?

Reply:
To be accurate, we should not count a span as `Accepted` if the metric is internal (like `count_per_op`). So, I think using the `exclusive_time` metric's conditions is the right move here and any condition since it's the broader condition of all the metrics we use in the UI.

If we ever have a metric used in the UI with a broader condition than this, we'd have to update `usage` metric's condition as well.

Reply:
Sounds good to me! Changed to `exclusive_time_light`, it's more lenient than `exclusive_time`.